stetho-sample/build.gradle

```diff
@@ -26,11 +26,17 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+
+    lintOptions {
+        // Rhino has references to awt and swing
+        disable 'InvalidPackage'
+    }
 }
 
 dependencies {
     debugCompile project(':stetho')
     debugCompile project(':stetho-urlconnection')
+    debugCompile project(':stetho-repl-js')
```

Comment:
I'll remove it. I have documentation on my sub project. I think that we can just leave the patch that affects the main README.md and to show users how to enable javascript by themselves.